Package.swift

```diff
@@ -55,7 +55,7 @@ let package = Package(
 		.package(url: "https://github.com/SwiftyJSON/SwiftyJSON", from: "5.0.2"),
 		
 		// Multicast / Share of notifications in EventBus
-		.package(url: "https://github.com/sideeffect-io/AsyncExtensions", exact: "0.5.3"),
+		.package(url: "https://github.com/sideeffect-io/AsyncExtensions", exact: "0.5.2"),
```

Comment:
was this intended? I think we want `0.5.3` for Xcode 16 support, but is it not Xcode 15 backwards compat? I think it is?

Reply:
this will be reverted once the draft is ready for review. Needed to drop down to `0.5.2` since Wallet is not compiling with `0.5.3`.